{"name": "test_login_3[2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <scripts.test_fixture.TestLogin object at 0x000001AA375CC6D8>, data = 2\n\n    def test_login_3(self, data):\n        \"\"\"记住密码登录\"\"\"\n        allure.attach(\"输入用户名\", \"输入用户名username_3\")\n        print(\"输入用户名\")\n        allure.attach(\"输入密码\", \"输入用户名password_3\")\n        print(\"输入密码\")\n        allure.attach(\"点击登录\", \"登录成功\")\n        print(\"点击登录\")\n        print(data)\n>       assert False\nE       assert False\n\nscripts\\test_fixture.py:40: AssertionError"}, "description": "记住密码登录", "attachments": [{"name": "输入用户名username_3", "source": "b21f5dcb-5dbc-4e0f-9d03-80c10e1cfb0d-attachment.attach"}, {"name": "输入用户名password_3", "source": "9bff55af-679c-4b61-9a79-459747f46b90-attachment.attach"}, {"name": "登录成功", "source": "8e5a9172-3a94-42ad-91dd-878ed6afb880-attachment.attach"}], "parameters": [{"name": "data", "value": "2"}], "start": 1575025910700, "stop": 1575025910710, "uuid": "ac6cf5dd-2497-45ad-85b7-9ebfe31abb22", "historyId": "3fa21cf732c1a3ca798649c5798ea1f0", "testCaseId": "133a8e3ccd283ba22e290157fe855949", "fullName": "scripts.test_fixture.TestLogin#test_login_3", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_fixture"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-47A72RD"}, {"name": "thread", "value": "8984-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_fixture"}]}